packages/babel-polyfill/package.json

```diff
@@ -21,5 +21,11 @@
   "dependencies": {
     "core-js": "^2.6.5",
     "regenerator-runtime": "^0.13.4"
+  },
+  "devDependencies": {
```

Comment:
They are pinned as the version that previously locked in `yarn.lock`. We can update them in subsequent PRs but given that `@babel/polyfill` is deprecated, this is of very low priority.

Reply:
`browserify` has been updated to `16.5.2` which bumps `browser-resolve` to 2.0.0 that supports Yarn PnP: https://github.com/browserify/browser-resolve/blob/master/CHANGELOG.md#browser-resolve-change-log